.github/workflows/run_tests.yml

```diff
@@ -1,7 +1,7 @@
 # This workflow will install Python dependencies, run tests and lint with a variety of Python versions
 # For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
 
-name: Run tests on all platforms
+name: CI/CD
```

Comment:
I believe a more descriptive name here would be more useful. Especially since we are running mostly tests and not doing delivery or "real" integration.

Reply:
That's true. There's no "D" in this script.

Reply:
Cool... let's go with the original & more descriptive name